DuckDuckGo/DuckPlayer/DuckPlayer.swift

```diff
@@ -216,8 +232,11 @@ final class DuckPlayer: DuckPlayerProtocol {
     }
 
     private func encodeUserValues() -> UserValues {
-        UserValues(
-            duckPlayerMode: settings.mode,
+        let experiment = DuckPlayerLaunchExperiment()
+        let isEnabled = experiment.isEnrolled && experiment.isExperimentCohort && featureFlagger.isFeatureOn(.duckPlayer)
```

Comment:
These values are the ones passed to the FE, so account for experiment group and feature flag